code/modules/mob/living/carbon/inventory.dm

```diff
@@ -140,3 +140,9 @@
 /mob/living/carbon/proc/get_holding_bodypart_of_item(obj/item/I)
 	var/index = get_held_index_of_item(I)
 	return index && hand_bodyparts[index]
+
+/mob/living/carbon/proc/get_internal_slots()
+	return list(
+		head,
+		wear_mask,

```

Comment:
This here will make most third-party byond dev tools cry. Here's an easy fix. Also there's no need for this to be multi-line, I'm not sure how this even compiles when you aren't cancelling out the linebreaks
```suggestion
		wear_mask
```